File: Python/uri_1040.py
n1, n2, n3, n4 = input().split(' ')
n1 = float(n1)
n2 = float(n2)
n3 = float(n3)
n4 = float(n4)
m = ((n1 * 2) + (n2 * 3) + (n3 * 4) + n4) / 10.0
print('Media: %0.1f' % m)
if m >= 7.0:
    print('Aluno aprovado.')
elif m < 5.0:
    print('Aluno reprovado.')
else:
    print('Aluno em exame.')
    n5 = float(input())
    print('Nota do exame: %0.1f' % n5)
    m = (m + n5) / 2.0
    if m >= 5.0:
        print('Aluno aprovado.')
    else:
        print('Aluno reprovado.')
    print('Media final: %0.1f' % m)

File: Go/uri_1040.go
package main

import "fmt"

func main() {
	var n [5] float64
	for i := 0; i < 4; i++ {
		fmt.Scan(&n[i])
	}
	med := (n[0]*2.0 + n[1]*3.0 + n[2]*4.0 + n[3]*1.0) / 10.0
	fmt.Printf("Media: %.1f\n", med)
	if med >= 7.0 {
		fmt.Printf("Aluno aprovado.\n")
	} else if med >= 5.0 && med <= 6.9 {
		fmt.Printf("Aluno em exame.\n")
		fmt.Scan(&n[4])
		fmt.Printf("Nota do exame: %.1f\n", n[4])
		med = (med + n[4]) / 2.0
		if med >= 5.0 {
			fmt.Printf("Aluno aprovado.\n")
		} else {
			fmt.Printf("Aluno reprovado.\n")
		}
		fmt.Printf("Media final: %.1f\n", med)
	} else {
		fmt.Printf("Aluno reprovado.\n")
	}

}
